zsh-syntax-highlighting-filetypes.plugin.zsh

```diff
@@ -0,0 +1 @@
+zsh-syntax-highlighting-filetypes.zsh
```

Comment:
I believe it should be 
```
source ${0:A:h}/zsh-syntax-highlighting-filetypes.zsh
```

Reply:
Hi @iceman201 --

You might be right! To be honest, I'm not using Oh My Zsh anymore but move to Prezto.

Feel free to make a new PR but I'm not sure it's going to be merged ever, mine was back in 2014...

Best to fork the repo and add your changes as I previously did.

Reply:
lol I didn't notice that was on 2014 until you mentioned 🤣